src/rules/examples-rule.ts

```diff
@@ -54,6 +55,20 @@ class ExamplesRule implements Rule {
         call.expectedResult,
         call.result
       );
+      if (!call.valid) {
+        // try to use ajv to check schema instead
+        try {
```

Comment:
might be worth putting this in a helper somewhere since we have similar code in a few spots now